### PR TITLE
Increment bower package version

### PR DESCRIPTION
**Pull request by codebeauty**
I updated version number from **0.6.14 to 0.6.15**, to release meta bugfix and added a small README for an better welcome on this bower repository.

@noraesae can you please review and create a  tag for 0.6.15 to get this small bugfix over bower. 

Thank you

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "perfect-scrollbar",
-  "version": "0.6.14",
+  "version": "0.6.15",
```
Comment (noraesae) [inline bower.json]: The version is kept the same with the original perfect-scrollbar version on npm. Can you please change it to something like `0.6.14-fix`?

**Comment by noraesae** [comment]
Thanks!

**Comment by noraesae** [comment]
I've just tested, and `-fix` tagging doesn't work well with bower. I just removed the original 0.6.14 tag and re-publish the current version as 0.6.14. Please clean cache and reinstall.

**Comment by codebeauty** [comment]
Good to know. Thank you.

**Pull request merged**